### ISSUE TITLE
SNV force calling

### ISSUE THREAD

**Issue by vollrath-p** [HKU-BAL/Clair#45]
Hello,
is there a way to perform SNV force calling using Clair?
E.g.: I did variant calling for 7 different genotypes. Afterwards I removed Indels and filtered for quality and AF ending up with 7 VCF files. After that, I merged the files using bcftools merge.
Now I want to use this VCF to do a force calling of this list of SNVs.
Many thanks,
Paul

**Comment by aquaskyline** [comment]
Is this option what you are looking for?
```bash
  --vcf_fn VCF_FN       Candidate sites VCF file input, if provided, variants
                        will only be called at the sites in the VCF file,
                        default: None
```

**Comment by vollrath-p** [comment]
Exactly. This was what I was looking for and what I already found. At first glance I thought it wouldn't work, but in combination with the information I got from closed issue #23, this produces the output I was aiming for. Thank you though.
